test/source/_posts/2015-02-20-extensionless-permalink.markdown

```diff
@@ -0,0 +1,7 @@
+---
+layout: ~
+title:  Extensionless Permalink
+permalink: /:title
+---
+
+Test
```

Comment:
Can we output `{{ page.url }}` here to ensure the Liquid URL is being set properly?